<html>
<head>
<head>
<body bgcolor="#000000"><p align="center">
<table width=700><tr><td>
<font color="#F7f7f7">
Readme for:
<p>
<a href="http://supramap.osu.edu/cov/janiesetal2008covsars.kmz">supramap.osu.edu/cov/janiesetal2008covsars.kmz</a>
</p><p>
This is an interactive visualization of the spread of SARS-CoV among various hosts, time, and geography.  It is supplemental data to the paper Janies, D., Habib, F. Alexandrov, B., Hill, A., and Pol, D. 2008. "Evolution of genomes, host shifts, and the geographic spread of SARS-CoV and related coronaviruses".  Cladistics. vol 23.  <a href="http://www.blackwell-synergy.com/toc/cla/0/0">http://www.blackwell-synergy.com/toc/cla/0/0</a>
 </p><p>
 Open the file <a href="http://supramap.osu.edu/cov/janiesetal2008covsars.kmz">janiesetal2008covsars.kmz</a> with Google Earth (available at <a href="http://earth.google.com">earth.google.com</a>).  
</p><p>
 The basic aspects of the visualization work well on any platform. However, currently the interactive features of the visualization work best with Windows XP or Vista, Google Earth 4.x, and Internet Explorer. 
</p><p>
 To make work between applications run smoothly, set your web browser to open KML with Google Earth by default and add http://www.supramap.osu.edu in the list of trusted sites. In some cases the internal Google Earth browser locks up after a few updates, but will always work if you open the 'View Flat Tree' link or 'Readme' link within Google Earth before executing any other links.
</p><p>
 Upon initially loading the file janiesetal2008covsars.kmz you will see branches representing a phylogeny based on genomic sequence data for 114 isolates of SARS CoV from diverse hosts and a non-SARS outgroup, Human Coronavirus NL63.
</p><p>
 The colors of the branches represent the host ancestry of SARS-CoV isolates.
</p><p>
 The ancestral nodes of the phylogeny are given some altitude via a procedure described in the paper: Janies, D., Hill, A., Guralnick, R., Habib, F., Waltari, E., Wheeler, W.C. 2007. Genomic Analysis and Geographic Visualization of the Spread of Avian Influenza (H5N1).  Systematic Biology.  56:321-9. <a href="http://www.informaworld.com/smpp/content?content=10.1080/10635150701266848">http://www.informaworld.com/smpp/content?content=10.1080/10635150701266848</a>
</p><p>
 The SARS-CoV viruses that were sequenced are assigned the geographic coordinates of the location of isolation. Each sequence shows up as an icon on the surface of the globe, whereas ancestral reconstructions are given a non-zero altitude. If you click on any icon you are presented with a pop-up window. In that pop-up window are several options. In the upper right of the pop-up you will see "View Flat Tree" option. By clicking on this you will load the ATV tree viewer, depicting a flat phylogeny of SARS-CoV.
</p><p>
 The user can click on nodes, branches, or isolates of interest in the flat phylogeny and be brought back to the same part of the phylogeny projected into Google Earth.
</p><p>
 The nucleotides on all positions have been optimized along the tree by maximum parsimony. Inferred mutations along a branch can be viewed in the Transformations table shown in the pop-up windows displayed when a node is clicked. The table lists the position, ancestral state, and descendant state for each nucleotide transformation along the branch from a node to its parent node.

</p><p>
<img src="http://supermap.osu.edu/cov114/desc_readme.png">
</p>
</font>
>/td></tr></table></p>
</body>
</html>
